.globalconfig

```diff
@@ -1460,10 +1460,10 @@ dotnet_diagnostic.SA1613.severity = none
 dotnet_diagnostic.SA1614.severity = none
 
 # SA1615: Element return value should be documented
-dotnet_diagnostic.SA1615.severity = none
+dotnet_diagnostic.SA1615.severity = suggestion
 
 # SA1616: Element return value documentation should have text
-dotnet_diagnostic.SA1616.severity = none
+dotnet_diagnostic.SA1616.severity = warning
 
 # SA1617: Void return value should not be documented
 dotnet_diagnostic.SA1617.severity = none
```

Comment:
It is not clear an intention of the PR. Why do we remove the doc tag if we need to fix it?
I'd expect:
dotnet_diagnostic.SA1615.severity = suggestion
dotnet_diagnostic.SA1616.severity =  suggestion
dotnet_diagnostic.SA1617.severity = warning

Reply:
No `returns` tag is better then an empty one, I've enabled `SA1617` and updated the PR description.

Reply:
We can't enable `SA1617` in this PR, because of the comments issue as expressed in #13978.

Reply:
It makes no sense to remove doc tags if we should have its and they should have values.